transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -12,6 +12,12 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
   /** (De)serializer for the upack messages we read from storage. */
   implicit val msgCoder: Coder[Msg] = Coder.beam(new UpackMsgCoder)
 
+  /** (De)serializer for the ODTs we extract from raw data. */
+  implicit val odtCoder: Coder[OffsetDateTime] = Coder.xmap(Coder.stringCoder)(
+    OffsetDateTime.parse(_),
+    _.toString
+  )
```

Comment:
Added an explicit coder to prevent falling back to Kryo, which results in an warning.